pkg/common/common.go

```diff
@@ -14,6 +14,8 @@ import (
 const (
 	// ReadUser is the global user for reading logs
 	ReadUser = "read"
+	// DefaultWriteTenant is the default tenant for writing logs
+	DefaultWriteTenant = "giantswarm"
```

Comment:
Aren't we renaming the default tenant to "Shared Org" with olly-op ? Or am I confusing things here ? :sweat_smile:

Reply:
Main Org will be renamed yes but not the tenant :D

Reply:
Ok my bad then :)